libs/pages/services/src/lib/ui/page-job-create/post/post.tsx

```diff
@@ -0,0 +1,240 @@
+import { BuildModeEnum } from 'qovery-typescript-axios'
+import { ServiceTypeEnum } from '@qovery/shared/enums'
+import { FlowVariableData, JobConfigureData, JobGeneralData, JobResourcesData } from '@qovery/shared/interfaces'
+import { Button, ButtonIcon, ButtonIconStyle, ButtonSize, ButtonStyle, Icon, IconAwesomeEnum } from '@qovery/shared/ui'
+
+export interface PostProps {
+  onSubmit: (withDeploy: boolean) => void
+  onPrevious: () => void
+  generalData: JobGeneralData
+  resourcesData: JobResourcesData
+  configureData: JobConfigureData
+  variableData: FlowVariableData
+  gotoGlobalInformation: () => void
+  gotoResources: () => void
+  gotoVariables: () => void
+  gotoConfigureJob: () => void
+  isLoadingCreate: boolean
+  isLoadingCreateAndDeploy: boolean
+  selectedRegistryName?: string
+  jobType: 'cron' | 'lifecycle'
+}
+
+export function Post(props: PostProps) {
```

Comment:
Same for UI components it's weird to have only Post you need to have the full name, like other creating flow